model/annotations/task_annotations.go

```diff
@@ -204,6 +204,57 @@ func UpdateAnnotation(a *TaskAnnotation, userDisplayName string) error {
 	return errors.Wrapf(err, "problem adding task annotation for '%s'", a.TaskId)
 }
 
+func AddToAnnotation(a *TaskAnnotation, userDisplayName string) error {
+	existingAnnotation, err := FindOneByTaskIdAndExecution(a.TaskId, a.TaskExecution)
+	if err != nil {
+		return errors.Wrapf(err, "finding task annotation for task id '%s'", a.TaskId)
+	}
+	if existingAnnotation == nil {
+		return errors.Wrapf(err, "missing task annotation for task id '%s'", a.TaskId)
```

Comment:
Why is this an error? If an annotation doesn't exist, do we not want to create one?

Reply:
Error is nil here so wrapping it still returns nil

Reply:
I think it would make sense to add an Upsert boolean to the route, where we insert a new annotation if one already exists (as this behavior may confuse new users or require them to make multiple calls).

Reply:
done